libs/langchain-google-common/src/utils/zod_to_gemini_parameters.ts

```diff
@@ -2,7 +2,37 @@
 
 import type { z } from "zod";
 import { zodToJsonSchema } from "zod-to-json-schema";
-import { GeminiFunctionSchema } from "../types.js";
+import { GeminiFunctionSchema, GeminiJsonSchema } from "../types.js";
+
+function removeAdditionalProperties(
+  schema: GeminiJsonSchema
+): GeminiJsonSchema {
+  const updatedSchema: GeminiJsonSchema = { ...schema };
+  if (typeof updatedSchema.additionalProperties === "boolean") {
```

Comment:
Why specifically test if its a boolean instead of just testing for its existence?

Reply:
Perhaps use `Object.hasOwn()`?
(I don't have a strong opinion on this. I just want to make sure we understand why we're testing it this way, since it seems less obvious.)

Reply:
Oh, I think that’s a better approach. Thanks for your help.